app/flows/check_uk_visa_flow/outcomes/outcome_work_m.erb

```diff
@@ -1,20 +1,40 @@
 <% text_for :title do %>
-  You'll need a visa to work, do business or academic research in the UK
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    You'll need to apply to do business, academic research or work in the UK
+  <% else %>
+    You'll need a visa to work, do business or academic research in the UK
+  <% end %>
 <% end %>
 
 <% govspeak_for :body do %>
-  The visa you need to apply for depends on your circumstances.
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    What you need to apply for depends on your circumstances.
+  <% else %>
+    The visa you need to apply for depends on your circumstances.
+  <% end %>
 
   <%= render partial: 'stateless_or_refugee', locals: {calculator: calculator} %>
 
   <%= render partial: 'estonia_or_latvia', locals: {calculator: calculator} %>
 
-  ##Business or academic visits
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    ##Business and academic visits
 
-  You can apply:
+    You can apply for:
+    + a [Standard Visitor visa](/standard-visitor) - if you’re coming to the UK for business activities like conferences, meetings, training, academic research or a sabbatical
+    + a [permitted paid engagement](/permitted-paid-engagement-visitor) visa if you’ve been invited as an expert in your profession - you can only stay for up to 1 month
 
-  + for a [Standard Visitor visa](/standard-visitor) - for example if you're coming to the UK for conferences, meetings, training, academic research or a sabbatical
-  + as a visitor doing a '[permitted paid engagement](/permitted-paid-engagement-visa)' (you must have been invited to the UK because of your expertise) - you can only stay for up to 1 month
+    ^
+    You will not need a visa if you hold a valid [electronic visa waiver (EVW)](/get-electronic-visa-waiver). You can apply for an EVW from 1 June 2022.
+    ^
```

Comment:
Change lines 27-29 from:

^
You will not need a visa if you hold a valid [electronic visa waiver (EVW)](https://github.com/get-electronic-visa-waiver). You can apply for an EVW from 1 June 2022.
^

to:

^You will not need a visa if you hold a valid [electronic visa waiver (EVW)](https://github.com/get-electronic-visa-waiver). You can apply for an EVW from 1 June 2022.^

Reason:

Markdown is incorrect - there should be no line break after the first ^ and before the second ^.